packages/common/src/search/_internal/hubSearchItemsHelpers/ogcItemToDiscussionPostResult.ts

```diff
@@ -23,11 +23,11 @@ export async function ogcItemToDiscussionPostResult(
     updatedDateSource: "properties.updatedAt",
     type: ogcItem.properties.postType,
```

Comment:
Let's change this to be a constant `type: 'post'` so functions down the line can use the field as a discriminator. Then functions that specialize in posts can access `rawResult.properties.postType` as needed`

Reply:
As you wish!